.github/workflows/pull-request.yml

```
@@ -312,3 +312,21 @@ jobs:
+      - name: Create .env file for talawa api testing environment
+        run: cp ./envFiles/.env.ci ./.env
+      - name: Build talawa api non production environment docker image
+        run: docker buildx build --file ./docker/api.Containerfile --tag talawa_api --target non_production ./
+      - name: Start Talawa API Container
+        run: docker compose up -d
+      - name: Import Sample Data
+        run: |
+          CONTAINER_ID=$(docker ps -q --filter "name=talawa-api-1")
+          docker exec -it $CONTAINER_ID /bin/bash -c "pnpm run import:sample-data"
+      - name: Stop Talawa API Container
+        run: docker compose down
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)